gulpfile.js

```diff
@@ -70,4 +74,21 @@ gulp.task('vendor-script', async () => {
         fs.mkdirSync(targetFolder);
     }
     await writeFileAsync(path.join(targetFolder, 'vendor.js'), result.code);
+});
+
+gulp.task('design-file', async () => {
+    let stylesdir = path.join(__dirname, './components/styles')
+    let content = '';
```

Comment:
Should we put a warning comment at the top that the file is auto generated?

Reply:
Will also see about adding a warning for design.css.